.travis-deploy.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+
+# Originally from https://gist.github.com/domenic/ec8b0fc8ab45f39403dd, with my
+# modifications for webpack.
+
+TARGET_BRANCH="pr3__bikeshed"
```

Comment:
This was all temporary experiments.  
I need to line up with @dontcallmedom to, most likely, restore the master branch and dump the compiled bikeshed to the gh-pages -- assuming, of course, that you go along with these changes, etc.